packages/eslint-config-worker/react.js

```diff
@@ -4,8 +4,12 @@ module.exports = {
 	plugins: ["eslint-plugin-react", "eslint-plugin-react-hooks"],
 	overrides: [
 		{
-			files: ["*.ts", "*.tsx"],
-			extends: ["plugin:react/recommended", "plugin:react-hooks/recommended"],
+			files: ["**/*.ts", "**/*.tsx"],
```

Comment:
Do we want to check `*.ts` for JSX?

Reply:
there might be hooks implemented in ts files, that can still be linted